### ISSUE TITLE
Incorrect vulnerability details: decompress@4.2.1 

### ISSUE THREAD

**Issue by EzeCuervo** [OSSIndex/vulns#191]
**Vulnerability URL**
```
https://ossindex.sonatype.org/component/pkg:npm/decompress@4.2.1
```

**Description**
The 4.2.1 decompress version fixed the known directory traversal vulnerability: 
```
https://github.com/kevva/decompress/releases/tag/v4.2.1
```

This is my first time reporting this kind of things, sorry if I forget something to add here.
Thanks!

**Comment by ken-duck** [comment]
Thank you very much for the heads up. The report was perfect. You should see the issue fixed in the public database sometime tomorrow.

**Comment by ndonewar** [comment]
Closing this issue as it appears to be resolved and version 4.2.1 is no longer included. Please feel free to reopen or create a new issue if needed. Thanks!
